rasa/nlu/featurizers/dummy_ner_featurizer.py

```diff
@@ -0,0 +1,69 @@
+import logging
```

Comment:
This file can be deleted, was just showing an example of the minimal requirement to create a Featurizer for this